src/bunit.core/Extensions/RenderedComponentInvokeAsyncExtension.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Text;
+using System.Threading.Tasks;
+using Bunit.Rendering;
+using Microsoft.AspNetCore.Components;
+
+namespace Bunit
+{
+	/// <summary>
+	/// InvokeAsync extensions methods on <see cref="IRenderedComponentBase{TComponent}"/>.
+	/// </summary>
+	public static class RenderedComponentInvokeAsyncExtensions
+	{
+		/// <summary>
+		/// Invokes the given <paramref name="callback"/> in the context of the associated <see cref="ITestRenderer"/>.
+		/// </summary>
+		/// <param name="renderedComponent">The rendered component whose dispatcher to invoke with.</param>
+		/// <param name="callback"></param>
+		/// <returns>A <see cref="Task"/> that will be completed when the action has finished executing.</returns>
```

Comment:
My suggestion: 
```suggestion
		/// <returns>A <see cref="Task"/> that will be completed when the action has finished executing or is suspended by an asynchronous operation.</returns>
```
Again from API consumer perspective: if I had read this instead I probably would have understood the problem more quickly.